setup.py

```diff
@@ -70,5 +70,6 @@
     keywords='ITK InsightToolkit segmentation registration image imaging',
     url=r'https://itk.org/',
     install_requires=[
+        r'numpy>=1.20',
```

Comment:
I'll follow-up, but we made sure that the typing support that we use in numpy is optional for systems that also want tensorflow.